land_duration: 10.0 # [s]

trajectory_sampling_rate: 10.0 # [Hz]

goto_destination:
  x: 0.0 # [m]
  y: 0.0 # [m]
  z: 1.0 # [m]
  heading: 0.5 # [m]
